phpstan.neon.dist

```diff
@@ -10,6 +10,19 @@ parameters:
             message: "#^Cannot call method (fetchOne|fetchAll|fetchAllAssociative|fetchAssociative|fetchAllKeyValue)\\(\\) on Doctrine\\\\DBAL\\\\ForwardCompatibility\\\\Result\\|int\\|string\\.$#"
             paths:
                 - tests/*
+        -
+            message: "#^Property Ibexa\\\\Contracts\\\\Core\\\\Repository\\\\Values\\\\Content\\\\Query\\\\Criterion\\:\\:\\$operator \\(string\\) does not accept string\\|null\\.$#"
```

Comment:
Cannot we do anything about those reports? They seem to be fixable at first glance.

Reply:
Fixed: https://github.com/ibexa/solr/pull/77/files#diff-be34c28f75856edd7547e70b50888be3da018caee383501c4a3b69edcbeb0216